.github/ISSUE_TEMPLATE/z-customer-feedback.yml

```diff
@@ -0,0 +1,67 @@
+name: Learn feedback control.
+description: |
+  ⛔ This template is hooked into the feedback control on the bottom of every page on the live site. It automatically fills in several fields for you. Don't use for other purposes. ⛔
+body:
+  - type: markdown
+    attributes:
+      value: "## Issue information"
+  - type: markdown
+    attributes:
+      value: Select the issue type, and describe the issue in the text box below. Add as much detail as needed to help us resolve the issue.
+  - type: dropdown
+    id: issue-type
+    attributes:
+      label: Type of issue
+      options:
+        - Typo
+        - Code doesn't work
+        - Missing information
+        - Outdated article
+        - Thank you
+        - Other (describe below)
+    validations:
+      required: true
+  - type: textarea
+    id: feedback
+    validations:
+      required: true
+    attributes:
+      label: Description
+  - type: markdown
+    attributes:
+      value: "## 🚧 Article information 🚧"
+  - type: markdown
+    attributes:
+      value: "*Don't modify the following fields*. They are automatically filled in for you. Doing so will disconnect your issue from the affected article. *Don't edit them*."
+  - type: input
+    id: pageUrl
+    validations:
+      required: true
+    attributes:
+      label: Page URL
+  - type: input
+    id: contentSourceUrl
+    validations:
+      required: true
+    attributes:
+      label: Content source URL
+  - type: input
+    id: documentVersionIndependentId
+    validations:
+      required: true
+    attributes:
+      label: Document Version Independent Id
+  - type: input
+    id: author
+    validations:
+      required: true
+    attributes:
+      label: Article author
+  - type: textarea
+    id: metadata
+    validations:
+      required: false
+    attributes:
+      label: Metadata
+      description: >-
+        Documentation metadata will be applied here for the PRMerger tool
```

Comment:
If the PR Merger tool isn't running in this repo, you can safely remove this section. Or leave it, if the info is valuable.